spec/controllers/teams_controller_spec.rb

```diff
@@ -6,31 +6,6 @@
   let(:team) { FactoryBot.create :team }
 
   describe 'GET #edit' do
-    context 'when signed in as a team leader without rights to manage all teams' do
```

Comment:
Same reason as mentioned here: https://github.com/thewca/worldcubeassociation.org/pull/9341/files#r1589365799.

Both the PRs need this test to be removed for all tests to pass.